src/lib/dnssd/minimal_mdns/core/tests/TestQName.cpp

```diff
@@ -40,299 +39,262 @@ static SerializedQNameIterator AsSerializedQName(const uint8_t (&v)[N])
     return SerializedQNameIterator(BytesRange(v, v + N - 1), v);
 }
 
-void IteratorTest(nlTestSuite * inSuite, void * inContext)
+TEST(TestQName, IteratorTest)
 {
     {
         static const uint8_t kOneItem[] = "\04test\00";
         SerializedQNameIterator it      = AsSerializedQName(kOneItem);
 
-        NL_TEST_ASSERT(inSuite, it.Next());
-        NL_TEST_ASSERT(inSuite, strcmp(it.Value(), "test") == 0);
-        NL_TEST_ASSERT(inSuite, !it.Next());
-        NL_TEST_ASSERT(inSuite, it.IsValid());
+        EXPECT_TRUE(it.Next());
+        EXPECT_EQ(strcmp(it.Value(), "test"), 0);
+        EXPECT_TRUE(!it.Next());
```

Comment:
Replace `EXPECT_TRUE(!...` with `EXPECT_FALSE`

Reply:
Done, now part of https://github.com/project-chip/connectedhomeip/pull/33069